pennylane/ops/qubit.py

```diff
@@ -1882,6 +1882,39 @@ def diagonalizing_gates(self):
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
 
+# =============================================================================
```

Comment:
Not a major thing, but I noticed that we broke out channels into a separate `channel.py`, worth considering here? I guess it depends how many projections we plan to add.